### PR TITLE
Add audit logs example application

### PR DESCRIPTION
**Pull request by awolfden**
Add audit logs example application and update SDK version on all apps

### CODE REVIEW

File: python-flask-audit-logs-example/app.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+# WorkOS Setup
+
+workos.api_key = os.getenv("WORKOS_API_KEY")
+workos.project_id = os.getenv("WORKOS_CLIENT_ID")
+workos.base_api_url = "http://localhost:7000/" if DEBUG else workos.base_api_url
+
+# Enter Connection ID here
+
+CUSTOMER_CONNECTION_ID = "conn_01G90XQ07953ZADRSFVF1YFSXP"
```
Comment (AshGodfrey) [inline python-flask-audit-logs-example/app.py:28]: small nitpick - do we want an actual connection ID here?

File: python-flask-audit-logs-example/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+   ```bash
+   # SSH
+   $ git clone git@github.com:workos/python-flask-example-applications.git
+   ```
+
+2. Navigate to the sso app within the cloned repo.
+
+   ```bash
+   $ cd python-flask-example-applications/python-audit-logs-example
```
Comment (AshGodfrey) [inline python-flask-audit-logs-example/README.md:28]: this path is incorrect (should be `python-flask-audit-logs-example`)

File: python-flask-audit-logs-example/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+Navigate to `localhost:5000`, or `localhost:5001` depending on which port you launched the server, in your web browser. You should see the WorkOS logo and a place to enter your Organization ID. The next steps occur in your WorkOS dashboard.
+
+You can stop the local Flask server for now by entering `CTRL + c` on the command line.
+
+## Audit Logs Setup with WorkOS
+
+11. Follow the [Audit Logs configuration steps](https://workos.com/docs/audit-logs/emit-an-audit-log-event/sign-in-to-your-workos-dashboard-account-and-configure-audit-log-event-schemas) to set up the following 5 events that are sent with this example:
+
+1. Action title: "user.signed_in" | Target type: "team"
```
Comment (AshGodfrey) [inline python-flask-audit-logs-example/README.md:113]: not sure why but all the steps after this are just "1" - we can probably update that

File: python-flask-audit-logs-example/app.py (hunk 1 above)
Comment (awolfden) [reply python-flask-audit-logs-example/app.py:28]: oh yeah thats just vestigial, removing it now

File: python-flask-audit-logs-example/README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Navigate to `localhost:5000`, or `localhost:5001` depending on which port you launched the server, in your web browser. You should see the WorkOS logo and a place to enter your Organization ID. The next steps occur in your WorkOS dashboard.
+
+You can stop the local Flask server for now by entering `CTRL + c` on the command line.
+
+## Audit Logs Setup with WorkOS
+
+10. Follow the [Audit Logs configuration steps](https://workos.com/docs/audit-logs/emit-an-audit-log-event/sign-in-to-your-workos-dashboard-account-and-configure-audit-log-event-schemas) to set up the following 5 events that are sent with this example:
+
+1. Action title: "user.signed_in" | Target type: "team"
```
Comment (AshGodfrey) [inline python-flask-audit-logs-example/README.md:111]: hmm, these numbers are still a bit odd - are they odd locally?

Comment (awolfden) [reply python-flask-audit-logs-example/README.md:111]: Its the auto-formatter... I just updated all of those and then they reverted. Let me see if I can get around it.

**Pull request merged**